### PR TITLE
[9.x] Introduce MissingAppKeyException for missing APP_KEY

### PR DESCRIPTION
**Pull request by bambamboole**
This PR introduces a new MissingAppKeyException.
It offers an easy way to catch this specific error.

I am building an application with an interactive installer and this should be the trigger for the setup.

### CODE REVIEW

File: src/Illuminate/Encryption/MissingAppKeyException.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Illuminate\Encryption;
+
+class MissingAppKeyException extends \RuntimeException
```
Comment (antonkomarev) [inline src/Illuminate/Encryption/MissingAppKeyException.php:5]: Import `\RuntimeException` class

**Comment by GrahamCampbell** [comment]
I think this would best be targetted at 8.x?

**Comment by driesvints** [comment]
We're not gonna merge too much anymore at this point.

**Comment by taylorotwell** [comment]
Send it to 8.x branch if you want it on 8.

**Pull request closed without merging**